sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -323,7 +323,8 @@ def run_pipeline(self, pipeline, options):
           'please install apache_beam[gcp]')
 
     # Convert all side inputs into a form acceptable to Dataflow.
-    if apiclient._use_fnapi(options):
+    if apiclient._use_fnapi(options) and (
+        not apiclient._use_unified_worker(options)):
```

Comment:
Why is this specifically excluded on the unified worker?

Reply:
Because this is making changes appropriate when using ISM files, but not (currently) when using the formats used by the unified worker.  We'll see if this should be refined when more features of side inputs are supported, and more efficiently, by the unified worker.